### DISCUSSION TITLE
Range items in loops

### DISCUSSION THREAD

**Discussion by jdvalderra** [atom/atom#22248] [Q&A]
Hey all, I'm new to coding, and I'm trying to fill out a square matrix of appending sequences. I can easily make vectors of appending sequences but concatenating them all is inefficient and time consuming. I was able to make it before but my Julia-Atom crashed with the 2 deprecated calls, and I thought the projects were saved to my computer but when I reset Atom they were lost. If I try to fill an array row with a range of numbers, say 1:15 it will work if I have something like:
B = zeros(Int, 15, 15) 
G = 1:15
B[1, :] = G
I get the first row as 
1 2 3 4 5...ect

But if I try it in a loop 
 For g = 1:15
B[1, :] = g
End 
 
I get the first row as 
15 15 15 15....

Can someone explain why this is happening? 
Any help is greatly appreciated, thanks.

**Comment by ThatXliner** [comment]
Are you using Juno? Also, can you show us the 2 deprecated calls?

**Reply by jdvalderra** [reply]
Sure, its 


atom core (2 deprecations)
Use require("electron").shell instead of require("shell")

Object.<anonymous> - C:\Users\jdval\AppData\Local\atom\app-1.56.0\resources\app.asar\exports\shell.js:4:6
Object.<anonymous> - C:\Users\jdval\AppData\Local\atom\app-1.56.0\resources\app.asar\exports\shell.js:9:3
Module._compile - <embedded>:11:149837
Object..js - <embedded>:11:153385
Module.load - internal/modules/cjs/loader.js:815:32
Module._load - internal/modules/cjs/loader.js:727:14
Use require("electron").remote instead of require("remote")

Object.<anonymous> - C:\Users\jdval\AppData\Local\atom\app-1.56.0\resources\app.asar\exports\remote.js:4:6
Object.<anonymous> - C:\Users\jdval\AppData\Local\atom\app-1.56.0\resources\app.asar\exports\remote.js:11:3
Module._compile - <embedded>:11:149837
Object..js - <embedded>:11:153385
Module.load - internal/modules/cjs/loader.js:815:32
Module._load - internal/modules/cjs/loader.js:727:14

**Reply by jdvalderra** [reply]
and yes I'm using Juno

**Reply by ThatXliner** [reply]
Aight. https://flight-manual.atom.io/faq/sections/i-m-having-a-problem-with-julia-what-do-i-do/

EDIT: I've seen those deprecated calls happening to people a lot recently. Problem came from the electron upgrade but I have Atom (1.56) too and I don't see those deprecated calls
